server/routes/nl_interface.py

```diff
@@ -147,7 +151,8 @@ def _chart_config(place_dcid, main_place_type, main_place_name,
       'place_dcid': [place_dcid],
   }
 
-  if child_places_type:
+  if (child_places_type and ('metadata' in chart_config) and
+      ('contained_place_types' in chart_config["metadata"])):
```

Comment:
nit: maybe try to keep it to only `'` or `"`?

Reply:
done, thanks